parse/match.go

```diff
@@ -1,19 +1,28 @@
 package parse
 
 import (
+	"regexp"
 	"strings"
 )
 
 func Match(pattern string, value string) (match bool, tokens []string) {
 	pattern = strings.ToLower(pattern)
 	ws := strings.HasPrefix(pattern, "*")
 	we := strings.HasSuffix(pattern, "*")
-	pattern = strings.Replace(pattern, "*", "", -1)
+	re := strings.HasPrefix(pattern, "/") && strings.HasSuffix(pattern, "/")
+	if re {
+		pattern = strings.Replace(pattern, "/", "", -1)
+	} else {
+		pattern = strings.Replace(pattern, "*", "", -1)
+	}
 	value = strings.TrimSpace(value)
 	tokens = strings.Split(strings.TrimSpace(strings.Replace(value, pattern, "", 1)), " ")
 	value = strings.ToLower(value)
 	match = false
-	if ws && ws && strings.Contains(value, pattern) {
+	if re {
+		regx := regexp.MustCompile(pattern)
+		match = regx.MatchString(value)
+	} else if ws && ws && strings.Contains(value, pattern) {
```

Comment:
This should be ws && we. Might as well change it now and close out the other issue I created for it!